src/test/scala/com/microsoft/ml/spark/core/test/fuzzing/FuzzingTest.scala

```diff
@@ -231,8 +234,17 @@ class FuzzingTest extends TestBase {
       "com.microsoft.ml.spark.lightgbm.LightGBMRegressionModel",
       "com.microsoft.ml.spark.vw.VowpalWabbitClassificationModel",
       "com.microsoft.ml.spark.vw.VowpalWabbitRegressionModel",
-      "com.microsoft.ml.spark.vw.VowpalWabbitContextualBanditModel"
+      "com.microsoft.ml.spark.vw.VowpalWabbitContextualBanditModel",
+      "com.microsoft.ml.spark.explainers.ImageLIME",
```

Comment:
These ones are about the HasInputCol mixins. I intentionally used the Spark's built-in `HasInputCol` trait, rather than mmlspark's override `com.microsoft.ml.spark.core.contracts.HasInputCol`. The difference is really just the `setInputCol` method, but I feel using Spark's built-in trait is safer, in the case where a customer may want to use the Spark's built-in `HasInputCol` trait to do pattern matching, or reflection.

Reply:
SG!